targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/rtl8195a_rom.h

```diff
@@ -0,0 +1,748 @@
+/*
```

Comment:
@0xc0170 @sg- it seems that they load binary ROM image with many symbols defined in it and then use them, that includes all ISR and a lot of standard library calls, are we ok with that?